### PR TITLE
Standardize Key Type annotation.

### LINKED ISSUE #3529: Make Key types consistent.
It seems in transformers we have used [Key](link) type and in Trainers we have used [key](link) type. Make it consistent everywhere by using [Key](link) type everywhere. 

CC: @Ivanidzo4ka  @shmoradims @natke

### PR DESCRIPTION
**Pull request by codemzs**
When referencing key types, use `[Key](xref:Microsoft.ML.Data.KeyDataViewType) type`.

fixes #3529

### CODE REVIEW

**Comment by Ivanidzo4ka** [comment]
I thought we chose other casing....

**Comment by codemzs** [comment]
@Ivanidzo4ka We chose this casing and what is important is keeping consistency. This PR’s title says that. 

CC: @shmoradims

File: docs/api-reference/io-columns-ranking.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+The input label data type must be [Key](xref:Microsoft.ML.Data.KeyDataViewType)
 type or <xref:System.Single>. The value of the label determines relevance, where
 higher values indicate higher relevance. If the label is a
-[key](xref:Microsoft.ML.Data.KeyDataViewType) type, then the key index is the
+[Key](xref:Microsoft.ML.Data.KeyDataViewType) type, then the key index is the
 relevance value, where the smallest index is the least relevant. If the label is a
 <xref:System.Single>, larger values indicate higher relevance. The feature
 column must be a known-sized vector of <xref:System.Single> and input row group
-column must be [key](xref:Microsoft.ML.Data.KeyDataViewType) type. This trainer
+column must be [Key](xref:Microsoft.ML.Data.KeyDataViewType) type. This trainer
```
Comment (shmoradims) [inline docs/api-reference/io-columns-ranking.md]: >column must be [Key](xref:Microsoft.ML.Data.KeyDataViewType) type.  [](start = 0, length = 67)

let's add a blank line here to separate input and output into two paragraphs:

column must be [Key](xref:Microsoft.ML.Data.KeyDataViewType) type. 
(blank line)
This trainer outputs ....

**Comment by shmoradims** [comment]
let use lower case `[key]` as discussed in the scrum.

**Pull request merged**